styles.scss

```diff
@@ -606,6 +606,48 @@ blockquote {
   }
 }
 
+.BioCheckbox {
+  grid-area: bioCheckbox;
+  input {
+    margin-right: 5px;
+  }
+}
+
+.CountrySelector {
+  grid-area: countrySelector;
+}
+
+.ImpactDisplay {
+  grid-area: impactDisplay;
+}
+
+.IngredientDelete {
+  grid-area: ingredientDelete;
+}
+
+.IngredientSelector {
+  grid-area: ingredientSelector;
+}
+
+.IngredientFormWrapper {
+  border-bottom: $list-group-border-width solid $list-group-border-color;
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: minmax(75px, 116px) 1fr 160px auto 120px 31px;
+  grid-template-areas: "massInputWrapper ingredientSelector countrySelector bioCheckbox impactDisplay ingredientDelete";
+  padding: $list-group-item-padding-y $list-group-item-padding-x;
+  @media screen and (max-width: $viewport-md) {
+    grid-template-columns: minmax(75px, 116px) 1fr 89px 31px;
+    grid-template-areas:
+      "massInputWrapper ingredientSelector impactDisplay impactDisplay"
+      "countrySelector countrySelector bioCheckbox ingredientDelete";
+  }
+}
```

Comment:
I'm really not sure about all this... From my tests it gives weird results at smaller viewports, which would be the whole purpose of using such a complex grid layout. In the screenshot blow, we can't even read the ingredient names. I'm pretty sure all this would be better achieved using good old bootstrap responsive cols. Also, I'm not sure the CSS grid layout here would be much more maintainable and debuggable in the future.

![image](https://user-images.githubusercontent.com/41547/212002117-849a14ce-c30b-4136-9143-08899407e7fc.png)

Reply:
What screen width is that? I tested with "bootstrap sm", and thought it was the smallest we were interested in "supporting"?

Reply:
It was 375px wide (iPhone SE, a *very* common phone). RGAA [states](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#10.11) a min of 320px...

Reply:
But don't spend too much time on this, we can live with the grid stuff as it is in the patch just fine, we can always refactor/update this later